src/bin/diff.js

```diff
@@ -94,3 +97,10 @@ function rulesDifference(a, b) {
     )
   );
 }
+
+process.exit(0);
+
+})().catch((e) => {
```

Comment:
@ljharb Should we test this? Or just ignore it? 🤔 E.g.,

```suggestion
})().catch(/* istanbul ignore next */(e) => {
```

Reply:
i think it's fine to ignore it.

Reply:
Done!